releasenotes/notes/wazuh-ansible-v4.10.0-ed5209199194cddf.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Upgrades the version of wazuh-ansible to v4.10.0. This brings in the SCA CIS
+    checks for Rocky Linux 9 by default.
```

Comment:
Does this mean we need to update the docs for custom SCA?

Reply:
Yes, good point